cms/publications/models.py

```diff
@@ -107,7 +110,14 @@ class PublicationPublicationTypeRelationship(models.Model):
     )
 
 
-class Publication(CategoryPage):
+class Publication(RoutablePageMixin, CategoryPage):
+    @route(r"^pdf/$")
+    def pdf_view(self, request):
```

Comment:
As discussed elsewhere -- may be worth catching if there's not a publication body available and 404ing or redirecting rather than rendering a bad PDF with an error message.